sympy/polys/rootisolation.py

```diff
@@ -2002,6 +2031,24 @@ def conjugate(self):
         return ComplexInterval(self.a, self.b, self.I, self.Q,
             self.F1, self.F2, self.f1, self.f2, self.dom, conj=True)
 
+    def __contains__(self, item):
+        """
+        Say whether a complex number belongs to this interval.
```

Comment:
Maybe this should clarify that it is a complex rectangular region. Is it common to use the word interval for this?

Reply:
Ah, good point. No, I haven't heard the word used this way outside this context.